src/rp2_common/hardware_irq/include/hardware/irq.h

```diff
@@ -258,6 +258,10 @@ irq_handler_t irq_get_vtable_handler(uint num);
 /*! \brief Clear a specific interrupt on the executing core
  *  \ingroup hardware_irq
  *
+ * This method is only useful for "software" IRQs that are not connected to hardware (i.e. IRQs 26-31)
```

Comment:
The `\param` part refers to `\ref interrupt_nums` and that only lists IRQs 0-25 ? (should that table also lists the software IRQs?)